external/os-guides/pentesting/kali.md

```diff
@@ -6,595 +6,168 @@ redirect_from:
 - /doc/kali/
 ---
 
-# How to create a Kali Linux VM
-
-## Warnings
-
-- The installation scripts and provided tools may have bugs, be vulnerable to Man in the Middle (MitM) attacks or other vulnerabilities.
-
-- Adding additional repositories or tools for installing software extends your trust to those tool providers.
-
-- Please keep in mind that using such a VM or VM's based on the template for security and privacy critical tasks is not recommended.
-
-- Kali Linux distribution is a rolling distribution based constantly on Debian testing release, so it always will have newer software base than available in Qubes OS debian template. Keep in mind that it may result in problems (especially in regard to package dependency) not covered by this tutorial.
-
-
-## Qubes 3.2
-
-### How to Create a Kali Linux VM
-
-This guide is being created to give guidance on ways in which you could create a [Kali Linux][kali] penetration testing VM (qube) in Qubes OS.
-
-Kali Linux is the most widely used penetration testing Linux distribution.
-
-There are multiple ways to create a Kali Linux VM:
-
- 1. Create a HVM and use the official ISO to install the system or convert a [Virtual Image][kali-vbox]. Explained [here](#hvm3_2).
- 2. Clone the Qubes OS latest Debian template image and turn it into a Kali Linux distribution:
-    - using [katoolin]. Explained [here](#katoolin3_2).
-    - manually. Explained [here](#templatevm-from-debian3_2).
-
-### Kali Linux HVM <a name="hvm3_2"/>
-
-1. Download the Kali installation DVD
-
+How to create a Kali Linux VM
+===============================
+Warnings
+--------------
+* The installation scripts and provided tools may have bugs, be vulnerable to Man in the Middle (MitM) attacks or other vulnerabilities.
+* Adding additional repositories or tools for installing software extends your trust to those tool providers.
+* Please keep in mind that using such template for security and privacy critical tasks is not recommended.
+* Kali Linux distribution is a rolling distribution constantly based on Debian testing release, so it will always have a newer software base than available in Qubes OS Debian template. Keep in mind that it may result in problems (especially in regard to package dependency) not covered by this tutorial.
```

Comment:
Kali Linux distribution is a rolling distribution based on Debian testing release, so it will always have a newer software base than available in the Qubes OS Debian template. Keep in mind that this may result in problems (especially in regard to package dependencies) not covered by this tutorial.